doc/source/dev/governance/governance.rst

```diff
@@ -378,6 +378,22 @@ A list of current Institutional Partners is maintained at the page
 :ref:`governance-people`.
 
 
+Commit Rights in the NumPy Code Base
+====================================
+
+The NumPy Steering Council can grant and revoke commit rights to the NumPy code
+repository, currently hosted at `github`_.
+
+The policy for granting commit priveleges is based on trust, with the
+understanding that they are needed for many of the mundane tasks required
+to maintain NumPy.  Upon receiving this priviledge, it is expected that the
+new committers, until they are experienced enough, will use it conservatively
+to triage issues, add labels, and merge only safe pull requests such as
+documentation.  Any significant code changes should be reviewed by at least
+two experienced NumPy contributors before merging.
+
+This policy may be revised at the discretion of the Steering Council.
```

Comment:
probably best to remove this, it doesn't add much and can be said about any technical or nontechnical decision about NumPy

Reply:
removed